packages/remark-plugins/all.js

```diff
@@ -12,6 +13,7 @@ export default function allPlugins({
   return [
     [includeMarkdown, includeMarkdownOptions],
     [anchorLinks, anchorLinksOptions],
+    jumpToSection,
```

Comment:
Thinking out loud....I wonder if we should exclude this one from the `all` list, only because it adds a net new UI element that we'll be phasing out sooner than later.

Reply:
Yea, good call and a I agree! I'll keep this out of `allPlugins`